gssapi/tests/test_raw.py

```diff
@@ -28,6 +33,45 @@ def setUpClass(cls):
 
         cls.USER_PRINC = cls.realm.user_princ.split('@')[0].encode("UTF-8")
         cls.ADMIN_PRINC = cls.realm.admin_princ.split('@')[0].encode("UTF-8")
+        cls.KRB5_LIB_PATH = cls._get_krb5_lib_path()
+
+    @classmethod
+    def _get_krb5_lib_path(cls) -> str:
+        if os.name == "darwin":
+            # FUTURE: Do we care if gssapi was compiled against something
+            # other than GSS.framework for the tests?
```

Comment:
You can compile against MIT (or even Heimdal on macOS) but it's a lot harder to do. You effectively need to set all the env vars and potentially need to make sure `krb5-config` is first in `PATH` but it's been a while since I last did that. For testing I just went the PATH of least resistance and always did the framework path as anyone wanting to test this code against something else probably knows how to change this (if they even run things).

Reply:
No longer needed as per https://github.com/pythongssapi/python-gssapi/pull/261#discussion_r685484611 the env var is set in the ci scripts and the test that requires this is just skipped if not present.